desktop/fileManager.js

```diff
@@ -42,6 +42,22 @@ class FileManager {
     }
   };
 
+  static showWalletBackupFile = async ({ event, fileName }) => {
```

Comment:
What the difference between this function and function some rows up?
Does highlighting file by name feature is good enough excuse to duplicate code?

Reply:
Purpose is to show the last backup. What do you suggest?

Reply:
What is the name of function above? And what is it's logic compared to the function you added? Can the previous function be modified to prevent code duplication?

Reply:
Got you. No problem.